frontend/packages/dev-console/src/components/topology/nodeContextMenu.tsx

```diff
@@ -48,8 +49,10 @@ const groupContextMenu = (element: Node) => {
 
   return createMenuItems(kebabOptionsToMenu(groupActions(applicationData)));
 };
-
 const nodeContextMenu = (element: Node) =>
   createMenuItems(kebabOptionsToMenu(nodeActions(element.getData())));
 
-export { workloadContextMenu, groupContextMenu, nodeContextMenu };
```

Comment:
Well this is an odd way to export out of this file

Reply:
@christianvogt 👀 Looking at you 😃

Reply:
Don't look at me... Jeff did it :P